beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -510,8 +511,17 @@ def _diagnostic_file_str_from_technique_partitions_post_conversion(self, techniq
         assert len(technique_partitions_post_conversion) > 0
         last_tech = technique_partitions_post_conversion[-1]
         num_techniques = last_tech.technique_num + 1 if last_tech.tech_does_loop else last_tech.technique_num
+        # Insert safety limits into the header for the file output
+        if self.min_voltage_v is None:
+            safety_min_v = "0 V"
+        else:
+            safety_min_v = str(self.min_voltage_v) + " V"
+        if self.max_voltage_v is None:
+            safety_max_v = "4.45 V"
```

Comment:
If we want defaults, why not just set them in the constructor and let the class throw if the user sets them to `None`?

Reply:
Done